include/onnxruntime/core/session/onnxruntime_c_api.h

```diff
@@ -129,7 +129,6 @@ extern "C" {
 
 // Used in *.cc files. Almost as same as ORT_API_STATUS, except without ORT_MUST_USE_RESULT and ORT_EXPORT
 #define ORT_API_STATUS_IMPL(NAME, ...) \
-  GSL_SUPPRESS(r .11)                  \
```

Comment:
maybe we should keep this, but if so we shouldn't have a GSL dependency in the C API header

Reply:
This macro is for internal uses only when implementing C API functions. So I'm less concerned.